zilencer/views.py

```diff
@@ -53,6 +53,19 @@ def remote_server_register_push(request, entity, user_id=REQ(),
 
     return json_success()
 
+@has_request_variables
+def remote_server_register_push_notification(request, hostname, contact_email, uuid, api_key):
+    remote_server, created = RemoteZulipServer.objects.get_or_create(
+        uuid=uuid,
+        api_key=api_key,
+        hostname=hostname,
+        contact_email=contact_email,
+        last_updated=timezone.now())
```

Comment:
Hmm -- I think this says that if there's already one with the right uuid, hostname, etc., but it has a different `last_updated` (which it always will), then we should try to create a new one. Is that right? That doesn't seem like what we mean.

Reply:
@gnprice fixed that